layouts/default.vue

```diff
@@ -64,7 +64,7 @@
                 <v-icon>mdi-account-key</v-icon>
               </v-list-item-icon>
               <v-list-item-content>
-                <v-list-item-title>Details</v-list-item-title>
+                <v-list-item-title @click="detail">Details</v-list-item-title>
```

Comment:
Since ```detail``` is a function, it would be better to rename the function. (e.g. **showDetails**)